CMakePresets.json

```diff
@@ -0,0 +1,237 @@
+{
+  "version": 6,
+  "include": [
+    "config/cmake-presets/hidden-presets.json"
+  ],
+  "configurePresets": [
+    {
+      "name": "ci-base-tgz",
+      "hidden": true,
+      "inherits": "ci-base",
+      "cacheVariables": {
+        "HDF5_ALLOW_EXTERNAL_SUPPORT": "NO",
+        "TGZPATH": {"type": "STRING", "value": "${sourceParentDir}/temp"}
+      }
+    },
+    {
+      "name": "ci-StdCompression",
+      "hidden": true,
+      "inherits": "ci-base-tgz",
+      "cacheVariables": {
+        "HDF5_ENABLE_Z_LIB_SUPPORT": "ON",
+        "HDF5_ENABLE_SZIP_SUPPORT": "ON",
+        "HDF5_ENABLE_SZIP_ENCODING": "ON",
+        "BUILD_ZLIB_WITH_FETCHCONTENT": "ON",
+        "ZLIB_PACKAGE_NAME": {"type": "STRING", "value": "zlib"},
+        "ZLIB_TGZ_ORIGPATH": {"type": "STRING", "value": "https://github.com/madler/zlib/releases/download/v1.2.13"},
+        "ZLIB_TGZ_ORIGNAME": {"type": "STRING", "value": "zlib-1.2.13.tar.gz"},
+        "ZLIB_USE_LOCALCONTENT": "OFF",
+        "BUILD_SZIP_WITH_FETCHCONTENT": "ON",
+        "LIBAEC_PACKAGE_NAME": {"type": "STRING", "value": "libaec"},
+        "LIBAEC_TGZ_ORIGPATH": {"type": "STRING", "value": "https://github.com/MathisRosenhauer/libaec/releases/download/v1.0.6"},
+        "LIBAEC_TGZ_ORIGNAME": {"type": "STRING", "value": "libaec-1.0.6.tar.gz"},
+        "LIBAEC_USE_LOCALCONTENT": "OFF"
+      }
+    },
+    {
+      "name": "ci-base-plugins",
+      "hidden": true,
+      "inherits": "ci-base-tgz",
+      "cacheVariables": {
+        "PLUGIN_TGZ_NAME": {"type": "STRING", "value": "hdf5_plugins-1.14.0.tar.gz"},
```

Comment:
@byrnHDF Does the plugins repo version get updated with each release? Is there a plugins-1.14.1?

Reply:
If we're just targeting the filter API, the maintenance branch (e.g., 1.14) should be enough.